index.js

```diff
@@ -2,29 +2,79 @@
  * Q0: Write a function called `countdown` that counts down from 10 on the console and
  * then logs "Blast Off!"
  */
+ function countdown() {
+ 	for (var i = 10; i >= 0; i--) {
+ 		console.log(i);
+ 		if (i <= 0) {
+	 		console.log('Blast off!!!');
+	 	}
+ 	}
+ }
+ countdown();
 
 
 /**
  * Q1: Write a function called `max` that has two Number parameters and returns
  * the larger of the two.
  */
+ function max(a, b) {
```

Comment:
This solution is nearly correct, except that the question asked us to `return` the value instead of logging it. When returning a value, we can only return 1, so we will have to pick one, even if they are equal. In the case of numbers, if they are equal, they are the same, so it doesn't matter:

```js
function max(a, b) {
    if (a > b) {
        return a;
    } else {
        return b;
    }
}
````